bin/ream/src/main.rs

```diff
@@ -77,11 +81,23 @@ async fn main() {
                 gossipsub_config,
             };
 
-            let ream_db = ReamDB::new(config.data_dir, config.ephemeral)
-                .expect("unable to init Ream Database");
+            let ream_dir = setup_data_dir(APP_NAME, config.data_dir.clone(), config.ephemeral)
```

Comment:
can you move the APP_NAME const to this crate, as it is no longer used in ream_storage right?